### PR TITLE
CaptionML snippet

### PR DESCRIPTION
**Pull request by kine**
I tried to add the CaptionML snippet to quickly create ENU and example of usage for more languages.

### CODE REVIEW

File: snippets/al.json [hunk 1]
```diff
@@ -262,7 +262,7 @@
     "Snippet: Caption ML": {
         "prefix": "tcaptionml",
         "body": [
-            "CaptionML = ENU = \"\"",
+            "CaptionML = ENU = \"\",",
```
Comment (StanislawStempin) [inline snippets/al.json]: Please use single quotes for the strings

File: snippets/al.json [hunk 2]
```diff
@@ -262,7 +262,7 @@
     "Snippet: Caption ML": {
         "prefix": "tcaptionml",
         "body": [
-            "CaptionML = ENU = \"\"",
```
Comment (StanislawStempin) [inline snippets/al.json]: Could you please add placeholders for the English text and the other text? That way, after adding the snippet the cursor will be placed at the right position and after tabbing will switch to the next placeholder.
Placheolder format is ${placeholderName}

Comment (StanislawStempin) [inline snippets/al.json]: Could you please add placeholders for the English text and the other text? That way, after adding the snippet the cursor will be placed at the right position and after tabbing will switch to the next placeholder.
Placheolder format is ${placeholderName}

Comment (StanislawStempin) [inline snippets/al.json]: Could you please add placeholders for the English text and the other text? That way, after adding the snippet the cursor will be placed at the right position and after tabbing will switch to the next placeholder.
Placheolder format is ${placeholderName}

File: snippets/al.json [hunk 3]
```diff
@@ -262,7 +262,7 @@
...
+            "CaptionML = ENU = \"\",",
             "\t\t\t\tXYZ = \"\";"
```
Comment (StanislawStempin) [inline snippets/al.json]: It will aligns better if you use 3x \t :)

**Comment by kine** [comment]
Done. Thanks for comments. How can I try the file on my installation before commiting it? Where I need to place it?

**Comment by StanislawStempin** [comment]
You can test it by placing the al.json file under:
c:\Users\<user name>\.vscode\extensions\Microsoft.al-0.0.1\snippets\

and restarting vscode

**Pull request merged**